modules/common/src/multi-jvm/scala/surge/internal/streams/StreamManagerMultiJvmSpec.scala

```diff
@@ -5,24 +5,24 @@ package surge.internal.streams
 import akka.NotUsed
 import akka.kafka.Subscriptions
 import akka.remote.testconductor.RoleName
-import akka.remote.testkit.{ MultiNodeConfig, MultiNodeSpec, MultiNodeSpecCallbacks }
+import akka.remote.testkit.{MultiNodeConfig, MultiNodeSpec, MultiNodeSpecCallbacks}
 import akka.stream.scaladsl.Flow
 import akka.testkit.TestProbe
-import com.typesafe.config.{ Config, ConfigFactory }
-import io.opentracing.Tracer
-import io.opentracing.noop.NoopTracerFactory
-import net.manub.embeddedkafka.{ EmbeddedKafka, EmbeddedKafkaConfig }
+import com.typesafe.config.{Config, ConfigFactory}
+import io.opentelemetry.api.trace.Tracer
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaConfig}
 import org.apache.kafka.clients.producer.ProducerRecord
-import org.apache.kafka.common.serialization.{ ByteArrayDeserializer, Deserializer, Serializer, StringDeserializer }
+import org.apache.kafka.common.serialization.{ByteArrayDeserializer, Deserializer, Serializer, StringDeserializer}
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpecLike
-import org.scalatest.{ BeforeAndAfterAll, OptionValues }
+import org.scalatest.{BeforeAndAfterAll, OptionValues}
 import surge.internal.akka.kafka.AkkaKafkaConsumer
+import surge.internal.tracing.NoopTracerFactory
 import surge.kafka.KafkaTopic
 import surge.kafka.streams.DefaultSerdes
-import surge.streams.replay.{ DefaultEventReplaySettings, KafkaForeverReplaySettings, KafkaForeverReplayStrategy, NoOpEventReplayStrategy }
-import surge.streams.{ DataHandler, EventPlusStreamMeta }
+import surge.streams.replay.{DefaultEventReplaySettings, KafkaForeverReplaySettings, KafkaForeverReplayStrategy, NoOpEventReplayStrategy}
+import surge.streams.{DataHandler, EventPlusStreamMeta}
```

Comment:
Hmm scalafmt should pick this up in the lint step but it must have missed it since it's in the multi-jvm scope. Should be easy enough to add that, I can make that change in a follow on PR but for now you can probably revert this file

Reply:
Fixed